README.md

```diff
@@ -8,6 +8,8 @@
 
 Loggly broadcasting for Good.
 
+`good-loggly` is a [good](https://github.com/hapijs/good) reporter implementation to write [hapi](http://hapijs.com/) server events to Loggly.
```

Comment:
It's really just a write stream to write things to loggly. I have been encouraging people to get out of the habit of tying themselves to good. You can stream anything to loggly with this module.